src/packagesjson.nim

```diff
@@ -1,5 +1,6 @@
 
 import std / [json, os, sets, strutils, httpclient, uri, options]
+import ./context
```

Comment:
```suggestion
import context
```

We don't use Status's "delusions of persecution" style.

Reply:
I didn’t realize it had a name 🤣